deploy/terraform-datacommons-website/examples/website_v1/main.tf

```diff
@@ -93,33 +94,21 @@ resource "google_compute_managed_ssl_certificate" "dc_website_cert" {
   }
 }
 
-data "google_container_cluster" "dc_web_cluster" {
-  name = module.cluster.name
-  location = var.region
-  project = var.project_id
-
-  depends_on = [module.cluster]
-}
-
-data "google_client_config" "default" {}
-
+# IMPORTANT NOTE: This script assumes that
+# "~/.kube/config" already exists. This is because provider cannot depend on data or resources,
```

Comment:
Maybe add some comments about what is "kube/config"?

Reply:
done.